### ISSUE TITLE
The meaning of bodyPeakPoint 

### ISSUE THREAD

**Issue by 1874ruanshaodong** [GitHub-Xuming/robot_dog#1]
Hi, what does bodyPeakPoint mean? Is it meaning that is the four vertices of the torso? Thanks.

**Comment by GitHub-Xuming** [comment]
Yes, that’s correct. If we regard the four legs as four mechanical arms, then “bodyPeakPoint” is the origin position of each mechanical arm.
